docs/fun.md

```diff
@@ -65,7 +65,7 @@ Name              | Example           | Usage
 <!-- tab:Prefix Commands -->
 Name              | Example           | Usage                                                                         
  ---------------- | ----------------- | ----------------------------------------------------------------------------- 
-**echo** [channel] \<message><br><span class="user-permissions">Manage Server</span> | `!echo #general Welcome guys` | Makes the bot post the message in the mentioned channel or the channel the command was used in.
+**echo** [channel] [no_button] \<message><br><span class="user-permissions">Manage Server</span> | `!echo #general Welcome guys` | Makes the bot post the message in the mentioned channel or the channel the command was used in. Premium users can toggle the button by setting `no_button` to either True or False. Toggling button is only enabled when a modlogs channel is set for the server. The button can only be pressed by users having Manage Server permission and it gives information of the user that used the command.
```

Comment:
Premium / Non premium can both remove the buttons now. we have removed the premium requirement

Reply:
buttons can be pressed by anyone but this will show the author to only those who have manage server perms.